src/py/aspen/database/models/sequences.py

```diff
@@ -180,6 +180,11 @@ class PathogenGenome(Entity):
         ),
     )
 
+    pangolin_lineage = Column(String)
+    pangolin_probability = Column(Integer)
```

Comment:
Should this be a float?

Reply:
i was looking at the `support` column here where it looked like it was an int, but should we be converting that to a float when we upload the table? 

<img width="1001" alt="Screen Shot 2021-04-20 at 5 13 08 PM" src="https://user-images.githubusercontent.com/13052132/115478768-b0341a80-a1fb-11eb-960b-033ccf636ae9.png">

Reply:
Oh, I think integer is fine then.  I think in the mocks, it was a float.